include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -1265,6 +1266,7 @@ def TT_ExperimentalDescriptorLoadOp : TT_Op<"experimental_descriptor_load", [Mem
   let arguments = (ins
     TT_TensorDescType:$desc,
     Variadic<I32>:$indices,
+    TT_DescriptorLoadAttr:$descAttr,
     DefaultValuedAttr<TT_CacheModifierAttr, "::mlir::triton::CacheModifier::NONE">:$cache,
     DefaultValuedAttr<TT_EvictionPolicyAttr, "::mlir::triton::EvictionPolicy::NORMAL">:$evict
```

Comment:
to enforce that they match I think we need to make it part of `TensorDescType`

Reply:
Some attributes are needed when creating `TensorDescType`. That's also why I separated such attributes needed for `MakeTensorDesc` from those needed during load.